From 02f57627e6bc213572485d3b7b1ffc73b3b66070 Mon Sep 17 00:00:00 2001
From: "Miouyouyou (Myy)" <myy@miouyouyou.fr>
Date: Sun, 5 Apr 2020 16:04:16 +0200
Subject: [PATCH 03/11] arm64: dts: rk3399-rockpro64: Doubling RXD DWMAC

Trying to mimick Ayufan's patch :
Ib3899f684188aa1ed1545717af004bba53fe0e07

Signed-off-by: Miouyouyou (Myy) <myy@miouyouyou.fr>
---
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
index 4b4271780..1e4a419fa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -28,3 +28,7 @@ es8316_p0_0: endpoint {
 		};
 	};
 };
+
+&gmac {
+	rx_delay = <0x20>;
+};
-- 
2.26.0

